crates/walrus-service/src/client/cli/runner.rs

```diff
@@ -761,12 +807,124 @@ impl ClientCommandRunner {
                     .join(", ")
             );
         }
-        tracing::info!(
-            duration = ?start_timer.elapsed(),
-            "{} out of {} blobs stored",
-            results.len(),
-            blobs_len
-        );
+
+        if !internal_run {
+            tracing::info!(
+                duration = ?start_timer.elapsed(),
+                "{} out of {} blobs stored",
+                results.len(),
+                blobs_len
+            );
+        }
+
+        if internal_run {
```

Comment:
Nit: this can be combined with previous if statement

Reply:
Good catch! will do